MetaMask/metamask-extension
# UX: Multichain: Move PickerNetwork to Wallet Tabs

## **Description**
The multichain effort requires that we move the `PickerNetwork` from the header to each home tab.  This PR moves the `PickerNetwork` to each of those tabs.  This PR does *not* account for the formatting of the header.  

## **Manual testing steps**

1.  If the `MULTICHAIN` flag is not enabled, nothing should change.
2. With the `MULTICHAIN` flag enabled, see no `PickerNetwork` in the unlocked state, regardless of full screen or popup
3. Clicking the `PickerNetwork` should launch the Network menu, and clicking an item should change the network
4. Test changing networks on each Wallet screen tab.

## **Screenshots/Recordings**

<img width="1318" alt="SCR-20231005-tjfg" src="https://github.com/MetaMask/metamask-extension/assets/46655/e6d3ec5a-2bb6-4221-bbd2-24d71d5b4d9a">



## **Related issues**

Fixes: https://app.zenhub.com/workspaces/metamask-extension-63529dce65cbb100265a3842/issues/gh/metamask/metamask-planning/683

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
